src/components/sub/menu/menu-item.jsx

```diff
@@ -1,23 +1,19 @@
-import React from 'react'
-import remixiconUrl from 'remixicon/fonts/remixicon.symbol.svg'
-import '@/components/sub/menu/styles.scss'
-import '@/components/sub/menu/menu-item.scss'
+import React from "react";
+import "@/components/sub/menu/styles.scss";
+import "@/components/sub/menu/menu-item.scss";
 
-
-
-export default ({
-  icon, title, action, isActive = null,
-}) => (
+const MenuItem = ({ children, title, action, isActive = null }) => (
```

Comment:
changing this from 

`export default ({ children, title, action, isActive = null }) => (`

to 

`
const MenuItem = ....

export default MenuItem
`

Is just a refactor and isn't the fix to the problem. I made the change bc its better to have named functions for debugging and error tracing. Its also easier for me personally to follow 

but again the way it was defined before would work